ocp_resources/virtual_machine.py

```diff
@@ -26,6 +26,14 @@ class RunStrategy:
         ALWAYS = "Always"
         RERUNONFAILURE = "RerunOnFailure"
 
+    class Status(NamespacedResource.Status):
```

Comment:
All the statuses should be under resource

Reply:
If you have a status that's specific for a card I do not think it should go under resource